projects/plugins/jetpack/extensions/blocks/anchor-fm/editor.js

```diff
@@ -75,11 +76,17 @@ const ConvertToAudio = () => {
 					'jetpack'
 				) }
 			</p>
-			<div role="link" tabIndex={ 0 } onClick={ handleClick } onKeyDown={ handleClick }>
-				<a href="https://anchor.fm/wordpressdotcom" target="_top">
-					{ __( 'Create a podcast episode', 'jetpack' ) }
+			<div
+				role="link"
+				className="post-publish-panel__postpublish-buttons"
+				tabIndex={ 0 }
+				onClick={ handleClick }
+				onKeyDown={ handleClick }
+			>
+				<Button isPrimary href="https://anchor.fm/wordpressdotcom" target="_top">
+					{ __( 'Create a podcast episode', 'jetpack' ) }{ ' ' }
```

Comment:
Would it be cleaner to do this space with left margin CSS on the icon instead?

Reply:
> Would it be cleaner to do this space with left margin CSS on the icon instead?

Agreed, this would be a nice follow-up.